src/http/app.ts

```diff
@@ -153,15 +155,21 @@ app.get("/data/:chainId/rounds/:roundId/vote_coefficients.csv", async (req, res)
   try {
     const [votes, data] = await Promise.all([
       db.collection(`rounds/${roundId}/votes`).all(),
-      fs.promises.readFile("./data/passport_scores.json", { encoding: "utf8", flag: "r" }),
+      fs.promises.readFile("./data/passport_valid_addresses.json", { encoding: "utf8", flag: "r" }),
```

Comment:
do you think it's useful to return the score as well in the CSV in addition to the coefficient being 1 or 0?